packages/plugin-core/src/test/suite-integ/GotoNote.test.ts

```diff
@@ -630,13 +633,44 @@ suite("GotoNote", function () {
           );
         },
         onInit: async () => {
-          // Open the note, select the hashtag, and use the command
           await WSUtils.openNote(note);
           VSCodeUtils.getActiveTextEditorOrThrow().selection =
             new vscode.Selection(7, 1, 7, 1);
 
           await new GotoNoteCommand().run();
-          // Make sure this took us to the tag note
+          // Make sure this took us to the file
+          expect(getActiveEditorBasename()).toEqual("test.txt");
+          expect(
+            VSCodeUtils.getActiveTextEditorOrThrow().document.getText().trim()
+          ).toEqual("Et voluptatem autem sunt.");
+          done();
+        },
+      });
+    });
+
+    test("non-note file, without slash in link", (done) => {
```

Comment:
is there a reason we're not using the new style workspace test here?

Reply:
This wouldn't have benefitted from the abilities of the new test style to share the same workspace across multiple tests, so I chose to go with the simpler option of copying and modifying existing tests. I'll go ahead and revise them though, you're right that we should still use the new style for new tests.